### PR TITLE
Added color, figures, and links

### PR DESCRIPTION
**Pull request by aThebigbot**
Hi, today I modified the npm user cli with color, figures and links. I hope you like my changes

### CODE REVIEW

**Comment by sindresorhus** [comment]
You could use https://github.com/sindresorhus/terminal-link to make the email and URLs clickable.

**Comment by sindresorhus** [comment]
CI is failing

**Comment by Thebigbot0000** [comment]
Now i fix all

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sindresorhus** [comment]
Did you see https://github.com/sindresorhus/npm-user-cli/pull/4#discussion_r863545381 ?

File: cli.js [hunk 1]
```diff
@@ -30,7 +32,13 @@ if (!username) {
 
 	const createRow = (prefix, key) => {
 		if (user[key]) {
-			rows.push(`${prefix}: ${user[key]}`);
+			if (key === 'github') {
+				rows.push(`${chalk.yellow(figures.star)} ${chalk.green(prefix)}: ${chalk.bold(user[key])} ${chalk.gray(`(${chalk.underline(`https://github.com/${user[key]}`)})`)}`);
+			} else if (key === 'twitter') {
+				rows.push(`${chalk.yellow(figures.star)} ${chalk.green(prefix)}: ${chalk.bold(user[key])} ${chalk.gray(`(${chalk.underline(`https://twitter.com/${user[key]}`)})`)}`);
+			} else {
+				rows.push(`${chalk.yellow(figures.star)} ${chalk.green(prefix)}: ${chalk.bold(user[key])}`);
```
[the remark this answers is not in the document]
Comment (Thebigbot0000) [reply cli.js]: Ok, but ava gives me an error if i inserted stars

**Comment by Thebigbot0000** [comment]
Ok, Asap i align the strings

**Comment by Thebigbot0000** [comment]
> Ok, Asap i align the strings

Excuse Me but i don't understand what should I do, all strings is align

**Comment by sindresorhus** [comment]
> Excuse Me but i don't understand what should I do, all strings is align

The alignment is different. It's pretty clear if you compare them side-by-side.

**Comment by sindresorhus** [comment]
The change in https://github.com/sindresorhus/npm-user-cli/pull/4/commits/85e0914cb8414d72ca697a8111b9c7ac6e392427 is clearly not correct.

**Comment by sindresorhus** [comment]
What I'm asking for in https://github.com/sindresorhus/npm-user-cli/pull/4#discussion_r863545381 is for the labels to be right-aligned, while their values are left aligned.

File: cli.js [hunk 2]
```diff
@@ -30,7 +32,13 @@ if (!username) {
 
 	const createRow = (prefix, key) => {
 		if (user[key]) {
```
Comment (sindresorhus) [inline cli.js:34]: I would recommend moving `user[key]` to a variable, since you are using it in many places.

File: cli.js [hunk 3]
```diff
@@ -30,7 +32,13 @@ if (!username) {
...
-			rows.push(`${prefix}: ${user[key]}`);
+			if (key === 'github') {
+				rows.push(`${chalk.green(prefix)}: ${chalk.bold("@"+user[key])} ${chalk.gray(terminalLink(`(Click Here)`, `https://github.com/${user[key]}`))}`);
```
Comment (sindresorhus) [inline cli.js:36]: The actual username should be the link. Not a "Click here" text.

Comment (Thebigbot0000) [reply cli.js:36]: I modify as soon as possible

**Comment by Thebigbot0000** [comment]
About #4 (comment) or fix as soon as possible

**Pull request closed without merging**